src/index.ts

```diff
@@ -11,24 +11,41 @@ export interface loginResponse {
     refresh: string
 }
 
-export interface AnalyzeOptions {
-    toolName?: string
-    noCacheLookup?: boolean
-    contractName?: string
-    bytecode?: string
-    sourceMap?: string
-    deployedBytecode?: string
-    deployedSourceMap?: string
-    mainSource?: string
-    sources?: any
-    sourceList?: Array<string>
-    solcVersion?: string
-    analysisMode?: string
+export interface VersionResponse {
```

Comment:
Many of the interfaces from this file can be made consistent with our pythx and other mythx schema using: https://www.npmjs.com/package/json-schema-to-typescript.

https://github.com/dmuhs/mythx-models/tree/master/mythx_models/request/schema
https://github.com/dmuhs/mythx-models/tree/master/mythx_models/response/schema

Reply:
ok I have a look into this